applications/FluidDynamicsApplication/custom_strategies/strategies/compressible_navier_stokes_explicit_solving_strategy_runge_kutta.h

```diff
@@ -167,7 +163,7 @@ class CompressibleNavierStokesExplicitSolvingStrategyRungeKutta
 
         // Shock capturing process check
         if (mShockCapturing) {
-            mpShockCapturingProcess->Check();
+            mpShockCapturingPhysicsBasedProcess->Check();
```

Comment:
I think that this should remain as `mpShockCapturingProcess`.

Reply:
Yep, I was too trigger-happy with Find and Replace.